templates/config.json

```diff
@@ -559,6 +588,10 @@
       "provider": "JupiterOne",
       "summary": "Security architecture is documented, including system and infrastructure security diagrams.",
       "guidance": "A good security architecture is the starting point of a strong program foundation. You will need to maintain an up to date architecture diagram for security reviews with customers and auditors. JupiterOne can help automate the generation of network and application architecture diagrams in your cloud environments, so you don't have to spend time maintaining and updating it every time something changes.",
+      "instructionsSummary": "Confirm/tailor infrastructure.",
+      "instructions": [
+        "What cloud service provider are you using? (AWS?)"
```

Comment:
something like "what cloud service provider + corresponding products/services are you using, e.g., AWS?"